projects/schematics/src/shared/utils/html-utils.ts

```diff
@@ -0,0 +1,123 @@
+import { SchematicsException, Tree } from '@angular-devkit/schematics';
```

Comment:
Can you also copy spec file for this utils?

Reply:
There's no spec file.

Reply:
https://github.com/angular/components/blob/master/src/cdk/schematics/utils/html-manipulation.ts